packages/api/lib/writeRecords/write-granules.js

```diff
@@ -569,9 +569,6 @@ const _writeGranule = async ({
     executionCumulusId,
     granulePgModel,
   });
-
-  // Reminder - this will only update the postgres files object
-  // if the granule is in a complete state
```

Comment:
I think the conflict was because my PR added a blank line here and master added this comment

https://github.com/nasa/cumulus/pull/3074/files#diff-705c5c617b45462fde4dd4c0b8b964f95e91ec857b774841daebe5144d9dd990